LanguageFeatures/Wildcards/unchanged_A05_t06.dart

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion Other declarations: top-level variables, top-level function
+/// names, type names, member names, etc. are unchanged. They can be named `_`
+/// as they are today.
+///
+/// @description Checks that it is a compile-time error to declare more than one
+/// member named `_`. Test a static variable vs other class members conflict.
+/// @author sgrekhov22@gmail.com
+
+// SharedOptions=--enable-experiment=wildcard-variables
+
+class C1 {
+  static int _ = 1;
+  static int _ = 1;
+//           ^
+// [analyzer] unspecified
+// [cfe] unspecified
+}
+
+class C2 {
+  static int _ = 1;
+  static int _() => 2;
+//           ^
+// [analyzer] unspecified
+// [cfe] unspecified
+}
+
+class C3 {
+  static int _ = 1;
+  static int get _ => 3;
+//               ^
+// [analyzer] unspecified
+// [cfe] unspecified
+}
+
+class C4 {
+  static int _ = 1;
+  static void set _(int v) {}
+//                ^
+// [analyzer] unspecified
+// [cfe] unspecified
+}
+
+class C5 {
```

Comment:
Perhaps create a variant?: `C5Base` could have the instance member and we could have `class C5 extends C5Base` or `class C5 implements C5Base`. The point would be to verify that the error is detected also in the case where the instance member isn't in the same lexical scope.